specs/gluster_cluster_list_view.adoc

```diff
@@ -0,0 +1,147 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of clusters discovered by Tendrl
+
+Display list of Gluster clusters present in Tendrl. The list should have functionality for 
+
+* filtering by name.
+* sorting by name, status, cluster version and managed.
+
+Cluster list will be sorted by name by default.
+
+View design will look like:
+
+image::upadtedClusterList.png[Upadted Cluster List]
+
+== Problem description
+
+Tendrl should show list of Gluster clusters present by Tendrl along with their details.
+
+The details shown for each cluster in UI is -
+
+* Cluster Name
+* Status
+* Managed State
+* Cluster Version
+* Import button(if ready to import)
+* Launch Dashboard button(if already imported)
+* Cluster Details button(if already imported)
+* Message to state cluster configurable state for eg. ready to import, cluster misconfigured.
+
+The workflow is as follows:
+
+* API will provide the list of clusters to the Tendrl.
+
+* User can click on expand icon of each cluster record to see the error messages/logs and hosts associated with it.
+
+* In the hosts tab, host name and IP address will be displayed.
+
+* Based on the managed state, import/launch dasboard/cluster details buttons will be displayed for each cluster.
+  * If Managed state is "Yes", Launch dashboard and cluster details button will be displayed.
+  * If Managed state is "No" and cluster is ready to import, Import button will be displayed.
+  * If Managed state is "No" and cluster is not properly configured, Imported will be disabled.
+
+* Clicking Import button will take to import cluster workflow.
+
+* Clicking Launch dashboard button will launch the dashboard in a new window.
```

Comment:
please clarify what you mean by new window. popup? or new tab in the browser? Also link with grafana dashboard spec